tests/itemCounter.test.js

```diff
@@ -0,0 +1,19 @@
+import itemcounter from '../src/modules/itemCounter.js';
+
+test('Should return correct count for items', () => {
+  const dummyData = [{ item: '1' }, { item: '2' }, { item: '3' }, { item: '4' }];
+
+  expect(itemcounter(dummyData)).toBe(4);
+});
+
+test('Should return correct count for items', () => {
+  const dummyData = [{ item: '1' }, { item: '2' }];
+
+  expect(itemcounter(dummyData)).toBe(2);
+});
+
+test('Should return correct count for items', () => {
+  const dummyData = [{ item: '1' }];
+
+  expect(itemcounter(dummyData)).toBe(1);
+});
```

Comment:
- [OPTIONAL] Kindly consider thinking about a scenario where there is 0 item on the page so that the itemCounter function should return 0. Kindly consider adding a test for this edge case.

Reply:
Implemented ✅✅. Thanks for the suggestions✌️.